coverage.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+# We are using an older version of the toolchain for now as there is a bug with
+# coverage generation. See https://github.com/rust-lang/rust/issues/79645
+export RUSTUP_TOOLCHAIN=nightly-2022-01-09
```

Comment:
I'm late to this, but I thought we were able to drop the date specific version of the toolchain?